README.md

```diff
@@ -1,29 +1,33 @@
-# GeoDev Meetup App
+# GeoDev Meetup/Hackerlab Apps
 
-This app provides three components:
+This repository contains all the apps for the GeoDev Meetup and Hackerlab events. The purpose of these apps is to ensure that we are effectively capturing attendees information.  There are four components:
 
-1. An Event Management page.
-2. An Attendee Registration page.
-3. An Attendee Raffle page.
+1. An Event Management app.
+2. A GeoDev Registration app.
+3. A GeoDev Name Picker app.
+4. An Attendee Viewer app. 
 
-## Event Management Page
-Use this page to create a new Feature that represents a meetup event.
-
-**This page doesn't exist yet :)**
+## Event Management
+Use this [Event Management app](http://edn1.esri.com/eventmanagement/index.html) to create a new Feature that represents a meetup or hackerlab event.
 
 This will collect the following information:
-* Location (use can use Search to find the address, or move the meetup on the map).
-* Name
-* Date
+* Event Name
+* Date (by default the timestamp is 12 am to ensure we record the event on the current date)
+* Event Type (e.g., meetup or hackerlab)
+* Voucher Code
+* Location (use can use the Search tab to find the address, the Lat/Lon tab or click on the map to add a point)
+
+Once you have created an event, you can register attendees using the GeoDev Registration app.
 
-Once you have created an event, you can register attendees using the Attendee Registration page.
+## GeoDev Registration
+The individual handling the check in will set the current event.
 
-## Attendee Registration Page
-The admin will set the current event.
+The [GeoDev Registration app](http://edn1.esri.com/meetup/register.html) will then allow people to enter their information (name, company, email, etc.) which will be written to a FeatureTable along with the key to the Meetup Feature.
 
-The app will then allow people to enter their information (name, company, email, etc.) which will be written to a FeatureTable along with the key to the Meetup Feature.
+Once all attendees have been registered, they can be considered for the giveaway using the GeoDev Name Picker.
 
-Once all attendees have been registered, they can be considered for the raffle using the Attendee Raffle page.
+## GeoDev Name Picker
```

Comment:
Minor addition - add that the name picker will allow you to select from the list of meetups created via the Event Management App